### PR TITLE
Add website/content/zh/docs/reference/glossary/sysctl.md.

### PR DESCRIPTION
**Pull request by ChongmingDu**
> For Chinese localization, base branch to release-1.14
Add website/content/zh/docs/reference/glossary/sysctl.md.

### CODE REVIEW

File: content/zh/docs/reference/glossary/sysctl.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: sysctl
+id: sysctl
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  用于获取和设置Unix内核参数的接口
```
Comment (chenrui333) [inline content/zh/docs/reference/glossary/sysctl.md]:
```suggestion
  用于获取和设置 Unix 内核参数的接口
```

File: content/zh/docs/reference/glossary/sysctl.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- tool
+---
+-->
+
+<!--
+ `sysctl` is a semi-standardized interface for reading or changing the
+ attributes of the running Unix kernel.
+-->
+ 
+ `sysctl`是一个半标准化的接口，用于读取或更改正在运行的Unix内核的属性。
```
Comment (chenrui333) [inline content/zh/docs/reference/glossary/sysctl.md]:
```suggestion
 `sysctl` 是一个半标准化的接口，用于读取或更改正在运行的 Unix 内核的属性。
```

File: content/zh/docs/reference/glossary/sysctl.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ 
+<!--more--> 
+
+<!--
+On Unix-like systems, `sysctl` is both the name of the tool that administrators
+use to view and modify these settings, and also the system call that the tool
+uses.
+-->
+
+在类Unix系统上，`sysctl`是管理员的工具,用于查看和修改这些设置，以及系统调用该工具使用。
```
Comment (chenrui333) [inline content/zh/docs/reference/glossary/sysctl.md]:
```suggestion
在类 Unix 系统上，`sysctl` 既是管理员的工具，也可以查看和修改这些设置，以及该工具所会用到的系统调用。
```

**Comment by ChongmingDu** [comment]
@chenrui333 I have revised it. Thanks.

**Comment by chenrui333** [comment]
Thanks for the update!

**Pull request merged**